back-end/cmp/celery.py

```diff
@@ -0,0 +1,7 @@
+import os
+from celery import Celery
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "cmp.settings.development")
```

Comment:
let the django settings module's last segment be fetched from the environment variable `SETTINGS` e.g 
```python
f"cmp.settings.{os.environ.get('SETTINGS')}"
```